src/common/admin_socket.cc

```diff
@@ -439,6 +440,166 @@ bool AdminSocket::do_accept()
   return rval;
 }
 
+namespace {
+template <typename Func>
+bool find_first_in(boost::string_view s, const char *delims, Func&& f)
```

Comment:
I think we can use std::string_view now that it's standardized in C++17

Reply:
@jcsp yes, but https://github.com/ceph/ceph/blob/master/src/include/str_list.h#L16 is still using its boost counterpart. i will do the switch over in another PR.